### ISSUE TITLE
Phoenix is not on the list of Web Frameworks on Wikipedia ... is it for real?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-phoenix-web-development#8]
At the time of writing, the Phoenix (Web) Framework is not in the list on Wikipedia:
https://en.wikipedia.org/wiki/Comparison_of_web_frameworks
Snapshot: [Comparison-of-web-frameworks-Wikipedia.pdf](https://github.com/dwyl/learn-phoenix-web-development/files/751425/Comparison-of-web-frameworks-Wikipedia.pdf)

The _reason_ this is an issue, is that non-technical decision makers will go looking for it and not find it in the comparison and conclude that it's not "a thing"... whereas there is no shortage of Java, PHP, JavaScript or Python frameworks, which people will _inevitably_ `default` to. 😞 

> If someone has time to update the Wikipedia article to reflect that both Elixir https://en.wikipedia.org/wiki/Elixir_(programming_language) and Phoenix (_which doesn't **appear** to have a page yet!_) are "a thing"!

**Comment by nelsonic** [comment]
have you seen this: http://hotframeworks.com ?
![image](https://cloud.githubusercontent.com/assets/194400/22681855/6d6366fa-ed07-11e6-91fe-e61f0c5e15d3.png)

I _predict_ that Phoenix will be "Number One" in 2018. We're catching the wave _early_!

**Comment by nelsonic** [comment]
Also: https://github.com/showcases/web-application-frameworks 
![image](https://cloud.githubusercontent.com/assets/194400/22682120/f2fdab76-ed08-11e6-9113-c9128e81b397.png)

**Comment by KelseyDH** [comment]
The answer to this problem is obvious and within Wikipedia's own guidelines:  Be bold and just create the damn page!

**Comment by iteles** [comment]
@KelseyDH That's exactly what this issue is here for. From the OP, this issue is so that: "If someone has time to update the Wikipedia article to reflect that both Elixir en.wikipedia.org/wiki/Elixir_(programming_language) and Phoenix (which doesn't appear to have a page yet!) are "a thing"!"

**Comment by nelsonic** [comment]
@iteles title updated.

**Comment by jayjun** [comment]
I’ve created a page for Phoenix, under review now. https://en.wikipedia.org/wiki/Draft:Phoenix

One moderator in a cursory glance said it’s unlikely to be approved because it references mostly primary sources. So feel free to chip in by citing other sources (that are not written by Chris McCord!).

**Comment by nelsonic** [comment]
@jayjun that's _amaze_! nice work! 🎉 
I don't see _why_ this would not pass moderation ... 🤔

**Comment by KelseyDH** [comment]
@jayjun  Nice work.  You'll likely need to rename the article to Phoenix Framework to get around the fact that Phoenix is currently [a disambiguation page](https://en.wikipedia.org/wiki/Phoenix), similar to how Scala's [Play Framework](https://en.wikipedia.org/wiki/Play_Framework) has [a disambiguation entry for the word play.](https://en.wikipedia.org/wiki/Play)

**Comment by KelseyDH** [comment]
All done!  [Enjoy.](https://en.wikipedia.org/wiki/Phoenix_(web_framework))

Sorry if I came off snarky earlier, after having my own draft pages languish in approval queues before...  I quickly learned that the zen to wikipedia editing is to skip these approval queues altogether... and just [be bold.](https://en.wikipedia.org/wiki/Wikipedia:Be_bold)

On that note, I've updated the [Phoenix disambiguation page](https://en.wikipedia.org/wiki/Phoenix#Computing) to include a link to this new page.

Enjoy! :)

**Comment by jayjun** [comment]
@KelseyDH Yes, my original intention all along was to rename it to `Phoenix (web framework)`, but was asked to do so only after approval.

By the way, the article just got declined. We’re close, just need to find enough reliable sources that are not written by the Phoenix team.

**Comment by KelseyDH** [comment]
@jayjun You're right it did get declined.  I resubmitted the article for approval, and I'm happy to report the [Phoenix (web framework) wikipedia page](https://en.wikipedia.org/wiki/Phoenix_(web_framework)) got approved and is now live!

**Comment by jayjun** [comment]
@KelseyDH Cool! Creating a new short article without rejection history was a great move. Hope it stays up.
